### PR TITLE
Example Solutions Numbering

### PR DESCRIPTION
**Pull request by LostMojo**
Adds an option to be able to number solutions within an example (as required for Statistics)

### CODE REVIEW

File: lib/kitchen/directions/bake_example.rb [hunk 1]
```diff
@@ -36,10 +36,15 @@ def self.v1(example:, number:, title_tag:)
           end
 
           if (solution = exercise.solution)
+            solution_number = if numbered_solutions
+                                "<span class=\"os-number\">#{exercise.count_in(:example)}</span>"
+                              else
+                                ''
+                              end
```
[the remark this answers is not in the document]
Comment (LostMojo) [reply lib/kitchen/directions/bake_example.rb:43]: it's a little too long to fit on one line

[the remark this answers is not in the document]
Comment (KarinaMendez2) [reply lib/kitchen/directions/bake_example.rb:43]: what about 
`
solution_number = ''
solution_number = "<span class=\"os-number\">#{exercise.count_in(:example)}</span>" if numbered_solutions

`

File: spec/directions/bake_example_spec.rb [hunk 2]
```diff
@@ -153,6 +153,97 @@
     end
   end
 
+  context 'when there is more than one exercise with solutions' do
+    let(:exercise) do
```
Comment (brittany-johnson) [inline spec/directions/bake_example_spec.rb:157]: I'm sorry if I'm missing the obvious but I don't see where `exercise` is being used?

**Review by brittany-johnson — CHANGES_REQUESTED** [review]

File: spec/directions/bake_example_spec.rb (hunk 2 above)
Comment (LostMojo) [reply spec/directions/bake_example_spec.rb:157]: it fits into :example above (so the example wrapper stays the same, but the content parts can get swapped around)

File: lib/kitchen/directions/bake_example.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (LostMojo) [reply lib/kitchen/directions/bake_example.rb:43]: well that's assigning a variable twice when I don't have to :T

**Review by brittany-johnson — APPROVED** [review]

**Pull request merged**